toolchain/mfc/test/test.py

```diff
@@ -192,6 +193,7 @@ def _handle_case(case: TestCase, devices: typing.Set[int]):
             if not os.path.exists(silo_filepath):
                 silo_filepath = os.path.join(case.get_dirpath(), 'silo_hdf5', 'p_all', 'p0', f'{t_step}.silo')
 
+            case.validate_params()
```

Comment:
Any particular reason for checking this here?

Reply:
None in particular -- will see if I can get away without it.